services/src/main/java/org/exoplatform/officeonline/WOPIService.java

```diff
@@ -609,7 +769,7 @@ protected boolean canUpdate(Node node) throws RepositoryException {
     try {
       parent = (NodeImpl) node.getParent();
     } catch (AccessDeniedException e) {
-      LOG.warn("Cannot get access to the parent node ", e);
+      LOG.warn("Cannot get access to the parent node ", e.getMessage());
```

Comment:
Error message after the comma? Or need concatenation with a first sentense?

Reply:
Fixed